.travis/build-deps.sh

```diff
@@ -3,7 +3,7 @@ set -eux
 
 # Set defaults
 MBEDTLS_VERSION="${MBEDTLS_VERSION:-2.2.1}"
-OPENSSL_VERSION="${OPENSSL_VERION:-1.0.2h}"
+OPENSSL_VERSION="${OPENSSL_VERSION:-1.0.1u}"
```

Comment:
as for moving to 1.0.2, I guess the best is to perform tests on the same version of SSL, which is used for release build:

https://github.com/OpenVPN/openvpn-build/blob/master/generic/build.vars#L4

we should probably change 1.0.1 --> 1.0.2 everwhere